src/components/HackerNews/HackerNews.tsx

```diff
@@ -0,0 +1,136 @@
+import * as React from "react";
+import { useToggleHackerNews, useHackerNewsFeed } from "@Store";
+import { useQuery, QueryClient, useQueries } from '@tanstack/react-query'
+import axios from 'axios';
+import { Card, LoaderCard } from "../Common/Card";
+import { z } from 'zod'
+import clsx from "clsx";
+
+const StoryIdSchema = z.number().array()
+
+export const HackerNews = () => {
+
+  // name == button name, title == window title, key == id, apiSrc == used in queries
+  // this can probably get cleaned up
+  const storyTypes = [
+    {
+      name: 'Top',
+      title: 'Top Stories',
+      key: 'TOP',
+      apiSrc: 'topstories'
+    },
+    {
+      name: 'Best',
+      title: 'Best Stories',
+      key: 'BEST',
+      apiSrc: 'beststories'
+    },
+    {
+      name: 'New',
+      title: 'New Stories',
+      key: 'NEW',
+      apiSrc: 'newstories'
+    },
+    {
+      name: 'Ask',
+      title: 'Ask Hacker News',
+      key: 'ASK',
+      apiSrc: 'askstories'
+    },
+    {
+      name: 'Show',
+      title: 'Show Hacker News',
+      key: 'SHOW',
+      apiSrc: 'showstories'
+    },
+    {
+      name: 'Jobs',
+      title: 'Jobs on Hacker News',
+      key: 'JOBS',
+      apiSrc: 'jobstories'
+    }
+  ]
+
+  const { setIsHackerNewsToggled } = useToggleHackerNews();
+  const { feed, setFeed } = useHackerNewsFeed();
+
+  console.log('feed', feed)
+
+  const selectedStoryType = storyTypes.find((storyType) => storyType.key === feed)
+
+  const { isLoading, error, data, refetch } = useQuery({
+    queryKey: [`storyList-${feed}`],
+    queryFn: async () => {
+
+      // let's grab the ids for the selected story type
+      const res = await axios.get(`https://hacker-news.firebaseio.com/v0/${selectedStoryType.apiSrc}.json`)
+        .then()
+
+      // parse for validation / TS and grab first 10
+      const ids = StoryIdSchema.parse(res.data).slice(0, 10);
+
+      // now grab the details for each story    
+      const data = await Promise.all(ids.map(async (id) => axios.get(`https://hacker-news.firebaseio.com/v0/item/${id}.json`)));
+
+      return data;
+    }
+  })
+
+  // trigger a refresh of the news stories
+  const refetchStories = async () => {
+    refetch()
```

Comment:
this is shooting refetches when other widgets are moved (see my comment in PR thread)

Reply:
There is no refetch happening. The component (as noted in the other comment) is rerendering like all components. Put a log on line 75 of HackerNews.tsx and you can confirm there is no refetch.